dll/win32/shlwapi/ordinal.c

```diff
@@ -3288,6 +3291,37 @@ BOOL WINAPI PlaySoundWrapW(LPCWSTR pszSound, HMODULE hmod, DWORD fdwSound)
 DWORD WINAPI SHGetIniStringW(LPCWSTR appName, LPCWSTR keyName, LPWSTR out,
         DWORD outLen, LPCWSTR filename)
 {
+#ifdef __REACTOS__
+    WCHAR szSection[MAX_PATH + 2];
+    WCHAR szBuffW[512];
+    CHAR szBuffA[512];
```

Comment:
512 is a hardcoded value. Is it justified? or is it possible to retrieve a more accurate length?

Reply:
Maybe use  a define with some speaking name for them at least?

Reply:
Fixed as `MAX_PATH`.